app/views/users/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<h1>User Profile</h1>
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th scope="col">id</th>
+      <th scope="col">Username</th>
+      <th scope="col">Email Address</th>
+      <th scope="col">Action</th>
+    
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>
+      <%= @user.id%> 
+      </th>
+      <th>
+      <%= @user.username%> 
+      </th>
+      <th>
+      <%= @user.email%> 
+      </th>
+      <td>
+      <%= link_to "Edit", edit_user_path%>
+
+     <%= link_to "Destroy", user_path(@user), data: {
+                    turbo_method: :delete,
+                    turbo_confirm: "Are you sure?"
+                  } %>
+      
+
+      </td>
+
```

Comment:
@MuhammadUsama1204 Please take care not to commit useless empty lines in your code.